docs/direct-access-urls.md

```diff
@@ -8,16 +8,21 @@ ___
 
 
 ## Overview
+Most Cloud Storage Providers allow generating publicly accessible URLs for sharing access to objects although the way it is provided and named differs throughout platforms.
+
 AWS S3 provides a way of generating
 [pre-signed URLs](https://docs.aws.amazon.com/AmazonS3/latest/dev/ShareObjectPreSignedURL.html)
-for sharing objects. This feature is a perfect candidate for the implementation of direct
+for sharing objects.
+Azure Blob Storage provide [Shared Access Signature (SAS) tokens](https://docs.microsoft.com/en-us/rest/api/storageservices/delegate-access-with-shared-access-signature) which as a part of object's URL can serve very similar role.
+
+Above described features are perfect candidates for the implementation of direct
 access to our content (e.g. shortcutting the Shared File Store for transformations or
 faster/direct access in the context of ADW to documents).
 
->:warning: **Note:** The AWS S3 pre-signed URLs are temporary links with an expiration time.
+>:warning: **Note:** The AWS S3 pre-signed URLs are temporary links with an expiration time. Same counts for Azure SAS tokens which have expiration time.
 
 The repository infrastructure now supports direct access urls. This includes the ContentService
-and the ContentStore interface for which default methods have been provided so that ContentStore implementations that 
+and the ContentStore interface for which default methods have been provided so that ContentStore implementations that
 implement the old version of this interface will throw a Not Supported exception.
 The new methods are auditable using the node reference and time in seconds for which the direct access URL is
 valid for as the  parameters.
```

Comment:
Trivial: reword "is valid for as the parameters." slightly (and remove extra space) ?

Reply:
Rephrased part is coming in the next commit.